src/Frontend/Core/Engine/Base/Block.php

```diff
@@ -137,6 +137,8 @@ public function addCSS(
         bool $minify = true,
         bool $addTimestamp = false
     ): void {
+        $overwritePath = $overwritePath || $this->get('fork.validator.url')->isExternalUrl($file);
```

Comment:
I don't find this very readable, I would at least add a comment which explains that external urls always override the path.

Reply:
fixed in 0ca9bdeb7